packages/roosterjs-editor-dom/lib/table/applyTableFormat.ts

```diff
@@ -148,8 +148,8 @@ function setCellFormat(
  */
 function setBorderColors(td: HTMLTableCellElement, format: Partial<TableFormat>) {
     td.style.borderTop = getBorderStyle(format.topBorderColor);
-    td.style.borderLeft = getBorderStyle(format.verticalBorderColor);
-    td.style.borderRight = getBorderStyle(format.verticalBorderColor);
+    td.style.borderInlineStart = getBorderStyle(format.verticalBorderColor);
```

Comment:
Suggest we fix content model editor only since bumping version of old package will cause additional cost but we don't see any benefit from it.

The old editor code will be deprecated.

Reply:
The issue is also present in the old version, but I agree that since it will be deprecated it is better to do it only in content model.``